### PR TITLE
Update checkpointing example to highlight Is_Resumable option

### PR DESCRIPTION
**Pull request by jhiemstrawisc**
Update README.md and checkpoint.sub to include information about accessing backfill

### CODE REVIEW

File: checkpointing/tensorflow/README.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ The example uses a pre-configured conda environment to demonstrate how one might
 
 The Tensorflow example downloads the MNIST dataset and trains a neural net on it for 20 epochs, making a "checkpoint" every fifth epoch. In the event of a runtime interruption, these checkpoints are preserved on the submit server and are used to resume training, preventing the loss of already-completed work.
 
-Checkpointing can be useful when running jobs that take multiple days to train or when training on resources that may be interrupted, like OSG GPUs.
+Checkpointing can be useful when running jobs that take multiple days to train or when training on resources that may be interrupted, like GPUs on CHTC backfill servers, campus pools outside of CHTC, or the Open Science Grid. These additional GPU resources can be accessed by adding the `Is_resumable = true`, `wantFlocking = true`, and `wantGlideIn = true` options, respectively, to your submit file.
```
Comment (agitter) [inline checkpointing/tensorflow/README.md:6]:
```suggestion
Checkpointing can be useful when running jobs that take multiple days to train or when training on resources that may be interrupted, like GPUs on CHTC backfill servers, campus pools outside of CHTC, or the Open Science Grid.
These additional GPU resources can be accessed by adding one or more of the following options to your submit file:
- `Is_resumable = true`: access CHTC backfill servers
- `wantFlocking = true`: access campus pools outside CHTC and CHTC backfill servers
- `wantGlideIn = true`: access the Open Science Grid, campus pools outside CHTC, and CHTC backfill servers
```
I like this, but what do you think about being more detailed about what each submit file line relates to?

@ChristinaLK are these descriptions accurate?

Comment (ChristinaLK) [reply checkpointing/tensorflow/README.md:6]: yes, I think so? I'm not 100% sure that `+WantGlidein` lets you also run on campus pools, but I can double check.

Comment (ChristinaLK) [reply checkpointing/tensorflow/README.md:6]: Also, we are now no longer the Open Science Grid, but just OSG, or "CHTC's OSG pool" (the king is dead, long live the king.)

Comment (agitter) [reply checkpointing/tensorflow/README.md:6]: I edited my suggestion above to use OSG instead.

**Review by agitter — APPROVED** [review]

**Review by ChristinaLK — APPROVED** [review]

**Pull request merged**